docs/live_update.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Optimizing a Tiltfile
```

Comment:
thought for future: if we want `LiveUpdate` to in face be a "simple case", maybe we should move this content into the main tutorial?